shared/Docs/Layout.tsx

```diff
@@ -98,11 +101,25 @@ export function Layout({
               <Header />
               <Navigation className="hidden lg:mt-6 lg:block" />
             </motion.header>
-            <div className="relative px-4 pt-14 sm:px-6 lg:px-8">
-              <main className="py-16">
+
+            {hidePageSidebar ? null : (
+              <motion.header
+                layoutScroll
+                className="fixed inset-y-0 my-14 pt-16 pb-12 right-0 z-40 hidden w-60 border-l border-slate-900/10 px-6 2xl:px-10 dark:border-white/10 xl:block 2xl:w-96"
+              >
+                <div className="pt-2">
+                  <PageSidebar />
+                </div>
+              </motion.header>
+            )}
+
+            <div className="relative px-4 pt-14 sm:px-6 lg:px-8 xl:pl-8 xl:pr-16 xl:mr-32 2xl:mr-10">
+              <main className="py-16 pr-20">
                 <Prose as="article">{children}</Prose>
               </main>
-              <Footer editPageURL={editPageURL} />
+              <div className={hidePageSidebar ? "" : "xl:pr-20 2xl:pr-80"}>
+                <Footer editPageURL={editPageURL} />
```

Comment:
Thank you! The solution was to move the `Footer` to be within the `Prose` content element which sets the responsive width. Because the `Prose` element sets some of its own spacing, I tweaked the `Footer` spacing to look nice. The `Footer` is now a little more condensed than it was before -- I think it looks better this way but open to feedback.